<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgColor</class>
 <widget class="QDialog" name="CQDlgColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Adjustment</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>RGB</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqRedValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqGreenValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqBlueValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="qqRedSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="qqGreenSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="qqBlueSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>HSL</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Hue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Luminosity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqHueValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqSaturationValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqLuminosityValue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="qqHueSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="qqSaturationSlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="qqLuminositySlider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="qqImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>61</width>
     <height>186</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>IMAGE</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqRedValue</tabstop>
  <tabstop>qqRedSlider</tabstop>
  <tabstop>qqGreenValue</tabstop>
  <tabstop>qqGreenSlider</tabstop>
  <tabstop>qqBlueValue</tabstop>
  <tabstop>qqBlueSlider</tabstop>
  <tabstop>qqHueValue</tabstop>
  <tabstop>qqHueSlider</tabstop>
  <tabstop>qqSaturationValue</tabstop>
  <tabstop>qqSaturationSlider</tabstop>
  <tabstop>qqLuminosityValue</tabstop>
  <tabstop>qqLuminositySlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
